tuxemon/technique.py

```diff
@@ -196,6 +197,7 @@ def load(self, slug: str) -> None:
                 logger.error(
                     f"Cannot find animation frames for: {self.animation}",
                 )
+        self.flip_axes = results.flip_axes
```

Comment:
You need to set flip_axes on the pydantic model within tuxemon/db.py 

Try Literal["x", "y, "xy"] as the typehint.

Reply:
Done.